Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml

```diff
@@ -107,36 +142,40 @@ tasks:
     isautoswitchedtoquietmode: false
   "15":
     id: "15"
-    taskid: 8dc00eec-3456-460b-8559-55713a961f59
+    taskid: cc2c2c53-ca98-46d9-8c20-77b6230387a4
     type: playbook
     task:
-      id: 8dc00eec-3456-460b-8559-55713a961f59
+      id: cc2c2c53-ca98-46d9-8c20-77b6230387a4
       version: -1
-      name: Detonate URL - Lastline v2
-      description: Detonates a URL using the Lastline sandbox integration.
-      playbookName: Detonate URL - Lastline v2
+      name: Detonate URL - FireEye AX
+      playbookName: Detonate URL - FireEye AX
       type: playbook
       iscommand: false
       brand: ""
+      description: ''
     nexttasks:
       '#none#':
-      - "3"
+      - "20"
     scriptarguments:
       URL:
         complex:
           root: inputs.URL
```

Comment:
add here uniq

Reply:
The 'unique' transformer already exists in the playbook input, are you sure I should add a 'unique' transformer to every sub-playbook?

Reply:
added